XamarinCommunityToolkit/Effects/Touch/TouchEffect.shared.cs

```diff
@@ -0,0 +1,1208 @@
+﻿using System.Windows.Input;
+using Xamarin.Forms;
+using System;
+using System.Linq;
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public class TouchEffect : RoutingEffect
```

Comment:
This class is very large. What do you think to split it into `partial classes`? We can name the file in a way that it's easy to find what it contains, like `TouchEffectProperties.shared.cs`, `TouchEffectTouchHandler.shared.cs` and so on.

Reply:
Do you really think it will help?) Almost all of these lines are related to API (define various properties)
if we will try to do it, TouchEffectProperties will contain 900/1000 lines of code 😄